plan_dinners.py

```diff
@@ -216,6 +216,12 @@ def export(self):
         self.meals_df.insert(0, f"Menu – Week of {self.week_timestamp}", [
                              "Meal", "Chef", "Ingredients", "Prep"], True)
 
+        """Clear Shopping sheet before refilling with new data"""
+        service = build("sheets", "v4", credentials=self.credentials)
```

Comment:
This should be moved into a PlanDinners.clear_sheet function like the update_sheet function, where it takes in the sheet and range info and clears it. It also needs to be correctly wrapped and catch http errors.